### PR TITLE
hidden fallback and jscs fixes

### LINKED ISSUE #86: input_type_hidden
Depend on https://github.com/bem/bem-components/issues/1379

### PR DESCRIPTION
**Pull request by verybigman**
#86

### CODE REVIEW

File: common.blocks/checkbox-group/checkbox-group.bh.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-module.exports = function (bh) {
+module.exports = function(bh) {
 
-    bh.match('checkbox-group', function (ctx, json) {
-	var _form_field = ctx.tParam('_form_field');
+    bh.match('checkbox-group', function(ctx, json) {
+        var _form_field = ctx.tParam('_form_field');
```
Comment (s-panferov) [inline common.blocks/checkbox-group/checkbox-group.bh.js]: Maybe it's better to refactor all these tab-spaces in another PR?

Comment (verybigman) [inline common.blocks/checkbox-group/checkbox-group.bh.js]: Maybe, but this is yesterday bug and this is not new functionality.

Comment (zxqfox) [inline common.blocks/checkbox-group/checkbox-group.bh.js]: Agreed that we need to fix them all.

```
$ grep '\t' ./common.blocks -r | cut -f 1 -d ':' | uniq
./common.blocks/checkbox/checkbox.bemhtml
./common.blocks/checkbox/checkbox.bh.js
./common.blocks/checkbox-group/checkbox-group.bemhtml
./common.blocks/checkbox-group/checkbox-group.bh.js
./common.blocks/form-field/form-field.bemhtml
./common.blocks/form-field/form-field.bh.js
./common.blocks/input/__control/input__control.bemhtml
./common.blocks/input/__control/input__control.bh.js
./common.blocks/radio/radio.bemhtml
./common.blocks/radio/radio.bh.js
./common.blocks/select/select.bemhtml
./common.blocks/select/select.bh.js
./common.blocks/textarea/textarea.bemhtml
./common.blocks/textarea/textarea.bh.js
```

**Comment by zxqfox** [comment]
:+1: Feels like there are not so many tabs. Can you look at the rest of them?

**Pull request merged**